blocks/heading/package.json

```diff
@@ -25,15 +25,15 @@
     "serve": "block-scripts serve"
   },
   "dependencies": {
-    "@blockprotocol/graph": "0.1.0",
-    "@blockprotocol/hook": "0.1.0"
+    "@blockprotocol/graph": "0.2.2",
+    "@blockprotocol/hook": "0.1.3"
   },
   "devDependencies": {
     "@local/eslint-config": "0.0.0-private",
     "@local/tsconfig": "0.0.0-private",
-    "block-scripts": "0.1.1",
+    "block-scripts": "0.2.3",
```

Comment:
Do you not need to fix (or rather introduce) the "codegen" object in these blocks?

Reply:
I will be doing this in immediate follow up because it will involve rewriting and moving all the type files, which will make for a large diff.

Reply:
<img width="1481" alt="image" src="https://user-images.githubusercontent.com/25749103/232067420-959f1049-80df-4a80-a1dd-e693bfee1e5b.png">

I didn't think you could get away with leaving it to another PR :(

Reply:
https://github.com/hashintel/hash/pull/2417/commits/674b9ed9e5cf6e3e78f6250b57f43132ba74c068